core/drivers/crypto/hisilicon/hpre_ecc.c

```diff
@@ -510,18 +553,14 @@ static enum hisi_drv_status hpre_ecc_parse_sqe(void *bd, void *info)
 		EMSG("HPRE do ecc fail! done=0x%"PRIX16", etype=0x%"PRIX16
 		     ",etype1=0x%"PRIX16, done, err, err1);
 
-		if (done == HPRE_HW_TASK_INIT) {
-			msg->result = HISI_QM_DRVCRYPT_ENOPROC;
```

Comment:
Why setting `msg->result` is not needed anymore? It looks like an unrelated change which should go in its own commit with explanations.

Reply:
we judge return value instead of msg->result, so msg->result is useless

Reply:
maybe I should use another pr to fix these two problem? These are not related to ecc sign and verify indeed.

Reply:
solved